docs/shared/deprecated/legacy-cache.md

```diff
@@ -0,0 +1,22 @@
+# Legacy Cache
+
+## tasksRunnerOptions
+
+As of Nx 20, the `tasksRunnerOptions` property in `nx.json` is deprecated. This property was used to register custom task runners. `tasksRunnerOptions` and custom task runners only work with the legacy file system cache.
+
+## NX_REJECT_UNKNOWN_LOCAL_CACHE
+
+The `NX_REJECT_UNKNOWN_LOCAL_CACHE` environment variable only works with the legacy file system local cache.
+
+## Next Steps
```

Comment:
Can we have 1 section which details the new cache.. and then have a separate section on next steps for Legacy cache users?

Reply:
It sounds like you're asking for a concept page

Reply:
I can move the content from the concept page into this page